lib/Zonemaster/LDNS/RR/DNAME.pm

```diff
@@ -17,6 +17,10 @@ A subclass of L<Zonemaster::LDNS::RR>, so it has all the methods of that class a
 
 =head1 METHODS
 
-No RDATA methods implemented yet.
+=over
 
-=cut
+=item dname()
+
+Returns the delegation name, i.e. the <target> field from the RDATA of a DNAME record.
```

Comment:
The RDATA only consists of the target so "from the RDATA" sounds strange. There is only one field to fetch. Maybe
"Returns the delegation name, i.e. the DNAME record target (RDATA)."